src/commands/monika.ts

```diff
@@ -69,9 +75,25 @@ export default class Monika extends Command {
   static flags = flags
   static id = 'monika'
 
+  async init(): Promise<void> {
+    await super.init()
+
+    if (flags.sentryDSN) {
+      initSentry(getContext().flags.sentryDSN as string)
+    }
```

Comment:
This will initialize sentry regardless since flags are not parsed yet.

```
INIT: Initializing sentry because flags.sentryDSN = "{"description":"DSN for Sentry","input":[],"multiple":false,"type":"option"}"
```

Reply:
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/8b830caf-8e43-49ce-bb7c-35f31ac371b0">
Resolved in the updated commit